### PR TITLE
Vote 2022-11-10

### PR DESCRIPTION
**Pull request by loga4**
Update allowed beacon balance increase limit to 2800

### CODE REVIEW

File: scripts/vote_2022_11_09.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+from utils.brownie_prelude import *
+from utils.config import (
+    get_deployer_account,
+    get_is_live,
+    contracts,
+)
+
+ALLOWED_BEACON_BALANCE_INCREASE_LIMIT = 2800
```
Comment (TheDZhon) [inline scripts/vote_2022_11_09.py:21]: should be `1750` according to the latest plan

File: scripts/vote_2022_11_09.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+"""
+Voting 09/11/2022.
```
Comment (TheDZhon) [inline scripts/vote_2022_11_09.py:2]: 10/11/2022

File: scripts/vote_2022_11_09.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    vote_items = bake_vote_items(vote_desc_items, call_script_items)
+    return confirm_vote_script(vote_items, silent) and create_vote(vote_items, tx_params)
+
+
+def main():
+    tx_params = {"from": get_deployer_account()}
+
+    if get_is_live():
+        tx_params["max_fee"] = "50 gwei"
```
Comment (TheDZhon) [inline scripts/vote_2022_11_09.py:52]: let's set with a safe margin: 200 gwei (extra expenses will not be charged due to the 1559 mechanics)

File: tests/test_2022_11_09.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import math
+from scripts.vote_2022_11_09 import start_vote
+from utils.test.tx_tracing_helpers import *
+
+ORACLE_ADDRESS = '0x442af784A788A5bd6F42A01Ebe9F287a871243fb'
+ALLOWED_BEACON_BALANCE_INCREASE_LIMIT = 2800
```
Comment (TheDZhon) [inline tests/test_2022_11_09.py:6]: 1750

File: tests/test_2022_11_09.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    # START VOTE
+    vote_id: int = vote_id_from_env or start_vote({"from": ldo_holder}, silent=True)[0]
+
+    tx: TransactionReceipt = helpers.execute_vote(
+        vote_id=vote_id, accounts=accounts, dao_voting=dao_voting, skip_time=3 * 60 * 60 * 24
+    )
+
+    # Validate vote events
+    if not bypass_events_decoding:
+        assert count_vote_items_by_events(tx, dao_voting) == 1, "Incorrect voting items count"
```
Comment (TheDZhon) [inline tests/test_2022_11_09.py:32]: let's check for the emitted `AllowedBeaconBalanceAnnualRelativeIncreaseSet` event :pray:

File: tests/test_oracle_report_under_heavy_mev.py [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        )
+
+        expectedEpoch = lido_oracle.getExpectedEpochId()
+        reporters = lido_oracle.getOracleMembers()
+        quorum = lido_oracle.getQuorum()
+
+        if days == 1:
+            expectedEpoch += epochsPerFrame * 4  # 3 days for the voting + 1 in for-loop
+
+        for reporter in reporters[:quorum]:
```
Comment (TheDZhon) [inline tests/test_oracle_report_under_heavy_mev.py:102]: may use some snippets to check for the actual rebase increase from here:
https://github.com/TheDZhon/sandwich/blob/main/tests/test_mev_tx_fee_sandwich.py#L115:L137

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM mostly, left some edits and suggestions :pray:

File: tests/test_2022_11_09.py (hunk 5 above)
Comment (loga4) [reply tests/test_2022_11_09.py]: +

**Pull request closed without merging**